verifiers/internal/gha/provenance.go

```diff
@@ -123,13 +125,17 @@ func verifySourceURI(prov slsaprovenance.Provenance, expectedSourceURI string, v
 	return nil
 }
 
-func sourceFromURI(uri string) (string, error) {
+// NOTE: `allowNoRef` is to allow for verification of npm packages
```

Comment:
should we set a tracking issue to remove this after GA?

Reply:
good idea. I was hesitant because I don't know that we'll be able to remove it right away, since some provenance will have been generated with this code. Unless all the provenance generated before GA is considered "no prod ready" and the npm teams removes it. Cerated https://github.com/slsa-framework/slsa-verifier/issues/524

Reply:
We can maybe only allow it for provenance created before a certain date? or something like that?

Reply:
Yeah, I wonder about the #524, and we can ask them when they go GA - i expect that is when they will stabilize